pylot/perception/detection/detection_operator.py

```diff
@@ -255,7 +255,8 @@ def __run_yolo_model(self, image_np):
         anchors = [float(x) for x in anchors.split(',')]
         anchors = np.array(anchors).reshape(-1, 2)
 
-        boxes, classes, scores = yolo3_postprocess_np(prediction, image_shape, anchors, 3, model_input_shape, elim_grid_sense=False)
+        num_classes = len(self._coco_labels)
+        boxes, classes, scores = yolo3_postprocess_np(prediction, image_shape, anchors, num_classes, model_input_shape, elim_grid_sense=False)
```

Comment:
Need to specify the number of classes based on the number of labels in the labels file.